src/main/java/org/folio/circulation/domain/representations/ItemProperties.java

```diff
@@ -4,6 +4,7 @@
   private ItemProperties() { }
 
   public static final String TITLE_PROPERTY = "title";
+  public static final String IDENTIFIERS_PROPERTY = "identifiers";
```

Comment:
Do you know why some of these constants have a `PROPERTY` postfix and others do not?

Reply:
No ideas. It worth renaming to `IDENTIFIERS_ID` as most of properties has ID postfix.

Reply:
I think `ID` should only be used if the property is an ID property (I think of that as part of the property name, not a postfix).

If the `PROPERTY` postfix can be removed from both without causing issues, that would be my preference